BlueprintUICommonControls/Sources/AccessibilityContainer.swift

```diff
@@ -0,0 +1,53 @@
+import BlueprintUI
+import UIKit
+
+/// Acts as an accessibility container for any subviews
+/// where `isAccessibilityElement == true`.
+public struct AccessibilityContainer: Element {
+
+    public var wrapped: Element
+
+    /// Creates a new `AccessibilityContainer` wrapping the provided element.
+    public init(wrapping element: Element) {
+        self.wrapped = element
+    }
+
+    //
+    // MARK: Element
+    //
+
+    public var content: ElementContent {
+        ElementContent(child: wrapped)
+    }
+
+    public func backingViewDescription(bounds: CGRect, subtreeExtent: CGRect?) -> ViewDescription? {
+        AccessibilityContainerView.describe { _ in }
+    }
+}
+
+public extension Element {
+
+    /// Acts as an accessibility container for any subviews
+    /// where `isAccessibilityElement == true`.
+    func accessibilityContainer() -> Element {
+        AccessibilityContainer(wrapping: self)
+    }
+}
+
+extension AccessibilityContainer {
+    private final class AccessibilityContainerView: UIView {
+        override var accessibilityElements: [Any]? {
+            get { recursiveSubviews().filter { $0.isAccessibilityElement } }
+            set { fatalError() }
+        }
+    }
+}
+
+extension UIView {
+    fileprivate func recursiveSubviews() -> [UIView] {
```

Comment:
Random Q: I forget how extensions on ObjC types work in Swift; do we need to prefix this like you would in ObjC land to avoid collisions?

Reply:
Should be safe if it's not `@objc`